dask/bag/text.py

```diff
@@ -92,6 +92,11 @@ def read_text(urlpath, blocksize=None, compression='infer',
         return from_delayed(blocks)
 
 
+def file_to_blocks(f):
+    with f as f:
+        return list(f)
```

Comment:
OK, seems to work. At some point you'll have to tell me more about fuse, when it gets called and how the multiprocessing takes place within execution that may be of a different scheduler.